### PR TITLE
PXP-548: [CLI] Program does not exit on Ctrl-C key press

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

The root cause of this is because we have 2 `ctrl-c` handlers, and they are conflicted with each other. I do a quick fix by removing the first `ctrl-c` handler (the one to show the cursor after `ctrl-c`) because I believe that unable to exit the cli with `ctrl-c` is much more annoying. 

Another solution I can think of is to use other key to close the instance connection, so we won't have the conflict at the first place. 

There are few ways to handle the signal too (https://tokio.rs/tokio/topics/shutdown), but this requires some refactoring. I think we can do this in the v2 implementation.

Ticket: [PXP-548](https://mindvalley.atlassian.net/browse/PXP-548)

### Fixed
- [x] fix the `ctrl-c` signal don't exit the cli issue

[PXP-548]: https://mindvalley.atlassian.net/browse/PXP-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main.rs
Comment (onimsha) [inline src/main.rs:13]: we should not leave commented code.

Comment (onimsha) [reply src/main.rs:13]: If this is a WIP, or a TODO, better put a clear comment section about what we want to do in the future.

**Review by onimsha — CHANGES_REQUESTED** [review]

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/112#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/112#pullrequestreview-1476123385" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
